docs/otlp/README.md

```diff
@@ -0,0 +1,31 @@
+# OpenTelemetry Protocol (OTLP)
+
+The [OpenTelemetry
+Specification](https://github.com/open-telemetry/opentelemetry-specification)
+defines the [OpenTelemetry Protocol
+(OTLP)](https://github.com/open-telemetry/opentelemetry-proto/tree/main/docs)
+which is a standard for exporting telemetry in a vendor agnostic way.
+OpenTelemetry .NET ships
+[OpenTelemetry.Exporter.OpenTelemetryProtocol](../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md)
+which is an implementation of OTLP and may be used to export telemetry to any
+backend with OTLP support or to the [OpenTelemetry
+Collector](https://github.com/open-telemetry/opentelemetry-collector).
+
+## Getting started
+
+If you are new to the [OpenTelemetry Protocol
+(OTLP)](https://github.com/open-telemetry/opentelemetry-proto/tree/main/docs),
+it is recommended to first follow the [getting started in 5 minutes - ASP.NET
+Core Application](./getting-started-aspnetcore/README.md) guide or the [getting
+started in 5 minutes - Console Application](./getting-started-console/README.md)
+guide to get up and running.
+
+## Vendor support
```

Comment:
FYI @alanwest @Kielek @martinjt

Reply:
Could we just link to website docs, instead of maintaining a list here : https://opentelemetry.io/ecosystem/vendors/

Reply:
I'm good with either. I think it is nice to link to specific docs for .NET, but I couldn't find one for everything. Also in that huge list not everything supports OTLP ingestion. I updated it to also link there though. LMK

Reply:
I'd prefer for this repo to not list and maintain this, and instead leverage docs website by linking to it.

If the website listing is not easy/hard-to-filter for .NET etc, then we can address that in the website